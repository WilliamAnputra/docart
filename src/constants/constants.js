export const ACCESS_TOKEN = 'accessToken';
export const PHOTOS = 'photo';