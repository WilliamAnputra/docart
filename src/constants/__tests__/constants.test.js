import React from 'react';


import * as constants from '../constants';

it('Renders constants ', () => {
    expect(constants).toMatchSnapshot();
});

