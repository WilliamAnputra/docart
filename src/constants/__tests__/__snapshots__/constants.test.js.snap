// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders constants  1`] = `
Object {
  "ACCESS_TOKEN": "accessToken",
  "PHOTOS": "photo",
}
`;
